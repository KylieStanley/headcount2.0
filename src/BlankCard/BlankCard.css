.blank {
  align-items: center;
  border: 6px double #008bc5cc;
  display: flex;
  justify-content: center;
}