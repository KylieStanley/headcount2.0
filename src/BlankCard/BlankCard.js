import React from 'react';
import './BlankCard.css'
import { library } from '@fortawesome/fontawesome-svg-core'
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
import { faAppleAlt } from '@fortawesome/free-solid-svg-icons'

library.add(faAppleAlt)

const BlankCard = () => {

  return (
    <div className="card blank">
      <h3>Select a School District</h3>
      <FontAwesomeIcon icon="apple-alt"/>
    </div>
  )
}

export default BlankCard;
