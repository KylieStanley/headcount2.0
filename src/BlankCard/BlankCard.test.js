import React from 'react';
import BlankCard from './BlankCard.js';
import {shallow, mount} from 'enzyme';

describe('BlankCard', () => {
  it('should match the snapshot with data passed in correctly', () => {
    const wrapper = shallow( <BlankCard />)

    expect(wrapper).toMatchSnapshot();
  })
})