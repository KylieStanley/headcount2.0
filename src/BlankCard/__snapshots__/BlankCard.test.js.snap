// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlankCard should match the snapshot with data passed in correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <BlankCard />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h3>
          School District
</h3>,
        <FontAwesomeIcon
          border={false}
          className=""
          fixedWidth={false}
          flip={null}
          icon="apple-alt"
          inverse={false}
          listItem={false}
          mask={null}
          pull={null}
          pulse={false}
          rotation={null}
          size={null}
          spin={false}
          symbol={false}
          title=""
          transform={null}
/>,
      ],
      "className": "card blank",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "School District",
        },
        "ref": null,
        "rendered": "School District",
        "type": "h3",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "border": false,
          "className": "",
          "fixedWidth": false,
          "flip": null,
          "icon": "apple-alt",
          "inverse": false,
          "listItem": false,
          "mask": null,
          "pull": null,
          "pulse": false,
          "rotation": null,
          "size": null,
          "spin": false,
          "symbol": false,
          "title": "",
          "transform": null,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h3>
            School District
</h3>,
          <FontAwesomeIcon
            border={false}
            className=""
            fixedWidth={false}
            flip={null}
            icon="apple-alt"
            inverse={false}
            listItem={false}
            mask={null}
            pull={null}
            pulse={false}
            rotation={null}
            size={null}
            spin={false}
            symbol={false}
            title=""
            transform={null}
/>,
        ],
        "className": "card blank",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "School District",
          },
          "ref": null,
          "rendered": "School District",
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "border": false,
            "className": "",
            "fixedWidth": false,
            "flip": null,
            "icon": "apple-alt",
            "inverse": false,
            "listItem": false,
            "mask": null,
            "pull": null,
            "pulse": false,
            "rotation": null,
            "size": null,
            "spin": false,
            "symbol": false,
            "title": "",
            "transform": null,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
